### DISCUSSION TITLE
How can i whitelist spawn worlguarded region outside of towny?

### DISCUSSION THREAD

**Discussion by DevilishDante** [TownyAdvanced/Towny#4866] [Q&A] [answered]
Hi, how can i out my spawn region of towny for player can interact or something else in her?
Despite i wanna stay towny enabled on the specified world
ty to read me ;)

**Comment by LlmDl** [comment, marked as answer]
You can protect your Spawn using an NPC town (read the Mayor section of the How Towny Works wikipage,) or use WorldGuard-Towny available from the website (read the readme on that repo.)

**Reply by DevilishDante** [reply]
i can make any npc town i want?

**Reply by DevilishDante** [reply]
can i create and claim some region like spawn without visit all the chunk? or isn't possible? 
Thanks you for your time :x

**Reply by LlmDl** [reply]
/t claim rect [a number representing the radius of the townblocks of the you want to claim]

or

/res toggle townclaim - and then fly around claiming any chunk you enter.
